shell/plugins/focus.js

```diff
@@ -39,3 +29,18 @@ const getElement = (vnode) => {
     return componentInstance.$refs.input.$refs.value;
   }
 };
+
+function focusDirective(_el, _binding, vnode) {
+  const element = getElement(vnode);
+
+  if (element) {
+    element.focus();
+  }
+}
+
+Vue.directive('focus', { inserted: focusDirective });
+
+export default focusDirective;
```

Comment:
I think that we should export an object that contains the hook functions, similar to how we export in `clean-html-directive`. This will enable us to register the directive without needing to understand anything about the internal implementation details (e.g.    `Vue.directive('intNumber', intNumberDirective)`, as opposed to `   Vue.directive('intNumber', { inserted: intNumberDirective })`). Additionally, we haven't tested the focus directive with other hooks like bind or update, so there's no guarantee that it will work properly with those.